packages-exp/auth-exp/src/core/user/token_manager.test.ts

```diff
@@ -222,7 +222,7 @@ describe('core/user/token_manager', () => {
 
         await expect(stsTokenManager.getToken(auth, true)).to.be.rejectedWith(
           FirebaseError,
-          'auth/internal-error'
+          'auth/token-refresh-unavailable'
```

Comment:
Apparently I can only comment on diffed lines in GitHub, but the error in line 266 should also be `token-refresh-unavailable` (also just noted this in your prior PR).

Reply:
Sick, updated!